src/content/language/functions.md

```diff
@@ -261,31 +261,30 @@ More about those in the next section.
 
 ## Anonymous functions
 
-Most functions are named, such as `main()` or `printElement()`.
-You can also create a nameless function
-called an _anonymous function_, or sometimes a _lambda_ or _closure_.
-You might assign an anonymous function to a variable so that,
-for example, you can add or remove it from a collection.
+Though you name most functions, such as `main()` or `printElement()`.
+you can create nameless functions.
```

Comment:
I feel like this clause is missing the "also" from the before version.

"...you can also create nameless functions."